### PR TITLE
Suppress pure-python SequenceMatcher warning

### PR DESCRIPTION
**Pull request by nanotubing**
Using the slow pure-python module isn't necessarily a problem for our use of this library, and this is a very noisy warning. We're suppressing this warning as we consider the implications of switching to a faster module in #2010 

## Notes

This PR suppresses only the warning:
"Using slow pure-python SequenceMatcher. Install python-Levenshtein to remove this warning"

This warning is thrown when thefuzz module is loaded, which is why the call to `warnings.filterwarnings` is so early in the file.

Connects #2009 

## Testing Instructions

Running `./scripts update` in `ogr/develop` will give output that looks like this, displaying the relevant warning a few times:
```
[+] Running 2/2
 ⠿ Container open-apparel-registry-database-1  Healthy                                                         0.6s
 ⠿ Container open-apparel-registry-django-1    Started                                                         0.9s
[+] Running 1/0
 ⠿ Container open-apparel-registry-database-1  Running                                                         0.0s
/usr/local/lib/python3.7/site-packages/thefuzz/fuzz.py:11: UserWarning: Using slow pure-python SequenceMatcher. Install python-Levenshtein to remove this warning
  warnings.warn('Using slow pure-python SequenceMatcher. Install python-Levenshtein to remove this warning')
Operations to perform:
  Apply all migrations: account, admin, api, auth, authtoken, contenttypes, sessions, sites, socialaccount, waffle
Running migrations:
  No migrations to apply.
```

* Check out this branch
* run `./scripts/update`. The output text will no longer display that warning.
```
[+] Running 2/2
 ⠿ Container open-apparel-registry-database-1  Healthy                                                         1.3s
 ⠿ Container open-apparel-registry-django-1    Started                                                         1.6s
[+] Running 1/0
 ⠿ Container open-apparel-registry-database-1  Running                                                         0.0s
Operations to perform:
  Apply all migrations: account, admin, api, auth, authtoken, contenttypes, sessions, sites, socialaccount, waffle
Running migrations:
  No migrations to apply.
```

## Checklist

- [x] `fixup!` commits have been squashed
- [x] CI passes after rebase
- [x] CHANGELOG.md updated with summary of features or fixes, following [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) guidelines
- [x] This PR is targeted at the correct branch (`develop` vs. `ogr/develop`)
- [ ] If this PR applies to both OAR and OGR a companion PR has been created

### CODE REVIEW

**Comment by jwalgran** [comment]
I suspect you could appease the linter by moving the call to `filterwarnings` below the rest of the import statements.

**Comment by nanotubing** [comment]
> I suspect you could appease the linter by moving the call to `filterwarnings` below the rest of the import statements.

I tried that first, but the warning filter wasn't taking affect until I put it at the top of the file, I think because the warning is thrown when thefuzz is imported.

File: src/django/api/facility_type_processing_type.py [hunk 1]
```diff
@@ -1,5 +1,10 @@
-from api.helpers import clean
-from thefuzz import process
+import warnings
+warnings.filterwarnings(
+    "ignore",
+    message="Using slow pure-python SequenceMatcher. Install python-Levenshte")
+
+from api.helpers import clean # NOQA
+from thefuzz import process # NOQA
```
Comment (maurizi) [inline src/django/api/facility_type_processing_type.py:7]: We can specify which lint rules specifically we want to opt out of, so we don't have to opt-out of lints we want to keep (e.g. "unused import")

```suggestion
from api.helpers import clean  # NOQA: E402
from thefuzz import process  # NOQA: E402
```

**Review by maurizi — APPROVED** [review]
This does suppress the warnings 👍 

I added a code suggestion but its pretty trivial, feel free to merge after addressing it w/o another review cycle.

**Pull request merged**